PerformanceNavigationTiming for User Agent Launch/security-privacy.md

```diff
@@ -58,7 +59,9 @@ No distinction.
 
 ## 2.14. How do the features in this specification work in the context of a browser’s Private Browsing or Incognito mode?
 
-There is a concern that exposing this information in an Private Browsing window could reveal that a user opened a certain website there. For example, if the user right mouse clicks on a link and selects “Open link in Incognito/InPrivate window”. However, such inference suffers from false positives, as the user may have invoked the URL and launched the browser from the OS shell, another non-browser application, or any other non-private browsing mode invocation.
+No distinction.
+
+There is a concern that exposing this information in an Private Browsing window, because it could indicate that a user visited a specific website there, since extensions are usually off by default in Private Browsing mode (thus eliminating one contributing factor). However, this inference is not reliable, because a 'low' confidence result can have many causes, and the result has noise added to it.

```

Comment:
```suggestion
There is a concern that exposing this information in a Private Browsing window could indicate that a user visited a specific website there, since extensions are usually off by default in Private Browsing mode (thus eliminating one contributing factor). However, this inference is not reliable, because a 'low' confidence result can have many causes, and the result has noise added to it.
```